src/plugins/intel_gpu/tests/functional/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "behavior/ov_infer_request/io_tensor.hpp"
 #include "openvino/runtime/properties.hpp"
+#include "ie_plugin_config.hpp"
```

Comment:
Is added for build purposes? Will this include be removed?

Reply:
yes, only to build fix, because this header is removed from more general header